src/content/pages/playground.html

```diff
@@ -5,6 +5,10 @@
 ---
 
 <div id="playground-app">
-    <p style="text-align:center;">run <span>npm run install:playground</span> before running playground</p>
+  <p style="text-align: center">
+    {% if helpers.isDev %} run <span>npm run install:playground</span> before
+    running playground
+  </p>
+  {% endif %}
```

Comment:
Let's move this whole `p` inside the if block.

```suggestion
  {% if helpers.isDev %} 
    <p style="text-align: center">
    run <span>npm run install:playground</span> before running playground
    </p>
  {% endif %}
```